packages/vm/src/dynamic_link.rs

```diff
@@ -425,7 +428,39 @@ mod tests {
             ));
 
             assert_eq!(result.err().unwrap().message(),
-            "func_info:{module_name:caller, name:foo, signature:[] -> []}, error:Unknown error during call into backend: Some(\"cannot found contract\")"
+            "func_info:{module_name:caller, name:foo, signature:[] -> []}, error:Error in dynamic link: Some(\"cannot found contract\")"
+            );
+        });
+    }
+
+    #[test]
+    fn dynamic_link_contract_fails() {
```

Comment:
Is this a test if the function does not exist?
How about clarify the name of the test as in test above(like `_not_exist_callee_`)?

Reply:
This is not a test that the function does not exist, but a test when panic occurs in callee of dynamic link.
The name is confusing, so I'll fix it.